ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+describe("Embed multiple form in site", () => {
+
+    context('Forms inside site', function () {
+  
+      before(() => {
+        cy.attachConsoleErrorSpy();
+      });
+      const pagePath = "/content/forms/sites/core-components-it/aemembedformstest.html";
+      let formContainer = [];
+
+      beforeEach(function () {
+          cy.previewForm(pagePath, {multipleEmbedContainers: true}).then(p => {
+              formContainer = p;
+          })
+      })
+  
+      it("should render three form", () => {
+        expect(formContainer.length).to.equal(1);
+      });
+  
+      it("form 1 model initialized properly", () => {
+          expect(formContainer[0], "formcontainer is initialized").to.not.be.null;
+          expect(formContainer[0].length).to.equal(5);
+      });
+    
+    it.skip("should submit on button click and thank you message of embed container must be overridden", () => {
+                cy.intercept({
+                method: 'POST',
+                url: '**/adobe/forms/af/submit/*',
+            }).as('afSubmission');
+            debugger;
+            cy.get(`.cmp-adaptiveform-button__widget`).eq(0).should('be.visible').wait(1000).click().then(x => {
+            cy.get('body').should('contain', "Thank You message from sites.\n")
+            });
+        });
+    it.skip("should submit on button click and thank you message of form should be visible not the embed", () => {
+            cy.intercept({
+            method: 'POST',
+            url: '**/adobe/forms/af/submit/*',
+        }).as('afSubmission');
+        cy.get(`.cmp-adaptiveform-button__widget`).eq(1).should('be.visible').wait(1000).click().then(x => {
```

Comment:
Let's not use wait with magic number

Reply:
This wait is required for button to get the response. I have changed it to 500 as suggested on the doc of cypress for the wait.